pkg/scaffold/util/util.go

```diff
@@ -26,6 +26,7 @@ import (
 	"sigs.k8s.io/kubebuilder/pkg/scaffold/resource"
 )
 
+// nolint:lll
```

Comment:
This can be fixed by either split the arguments across multiple lines or split the return values across multiple lines.